docs/source/_templates/layout.html

```diff
@@ -9,23 +9,23 @@
   }}();
 </script> -->
 
-<title>EvalML - Alteryx Open Source AutoML Library</title>
-<meta name="description" content="EvalML is an AutoML library that builds, optimizes, and evaluates machine learning pipelines using domain-specific objective functions. — EvalML documentation">
+<title>CheckMate - Alteryx Open Source AutoML Library</title>
```

Comment:
`Alteryx Open Source Data Checks for Machine Learning Library` or something similar

Reply:
refactored